lib/options.js

```diff
@@ -386,6 +386,19 @@ module.exports = {
         usage: ['CONVERSION'],
         supportedIn: [VERSION20, VERSION30, VERSION31],
         supportedModuleVersion: [MODULE_VERSION.V2, MODULE_VERSION.V1]
+      },
+      {
+        name: 'Select request body type',
+        id: 'preferredRequestBodyType',
+        type: 'enum',
+        default: 'x-www-form-urlencoded',
+        availableOptions: ['x-www-form-urlencoded', 'form-data', 'raw'],
+        description: 'When there are multiple content-types defined in the request body of OpenAPI, select the ' +
+          'preferred option content-type as request body.',
+        external: true,
```

Comment:
This can be false for CLI only usage.

Reply:
changed to `external: false`

Reply:
@thim81 Not really, below is screenshot of exposed options in Postman. i.e. `requestNameSource` is set as `external: true` and is displayed on app.

![image](https://github.com/user-attachments/assets/504f6802-55c6-4bd7-a232-33089880ede5)

As for CLI, we don't really consider `external` value. See screenshot for below.

![image](https://github.com/user-attachments/assets/2d5110de-8166-4605-8314-a5e8f3991266)

Reply:
I set `external:false`, the structure.test.js `OPTIONS.MD`>`must contain all details of options` fails, it expects only the "external:true" items? 

This would mean remove the option from the OPTIONS.MD which would be strange, since the option.md references all options (CLI & non-CLI). I think the test should not filter out the external:false options?

Reply:
@thim81 Got it. Yeah, we had set the OPTIONS.md to not include info about `external:false` items previously.
But now they can be present. In that case, we can also update this test and include external column into OPTIONS.md for all options as well.

Let me know if you wish to work on this? I can also take that up if you like and raise separate PR.

Reply:
@VShingala I updated the options.md and the related test.
Is this what you were expecting?